ui/webui/test/README.rst

```diff
@@ -1,17 +1,46 @@
-Integration Tests of Anaconda WebUI
-===================================
+Anaconda Web UI tests
+=====================
 
-This directory contains automated integration tests for Anaconda WebUI, and the support files for them.
+The Web UI tests are based on cockpit's testing framework which in turn is based
+on python's unittest library. There are two kinds of Web UI tests, "integration"
+that test only one specific feature at a time and "end to end" that go through the
+entire installation process including reboot to the installed system. Both types
+use the same library of helper functions, to avoid duplicating code and to make
```

Comment:
'Both use the same library of helper functions'

This rephrases the first sentence, which says that both are based on cockpit's testing framework.

Additionally I actually liked the introduction line that you removed.

Generally try to avoid too long sentences as it heavily harms readability. This paragraph had for example ~10% readability rating in https://textinspector.com/index.php, and I believe it's mostly because of the many secondary sentences.

Here is my suggestion: 


```
This directory contains automated integration and end to end tests for Anaconda WebUI, and the support files for them.
Integration tests focus on individual features, addressing one specific functionality at a time. On the other hand, end-to-end tests encompass the complete installation process, including a reboot to validate the functionality of the installed system.
Both test categories utilize Cockpit's testing framework. However, they are executed differently due to the heightened demands of the end-to-end tests.
```

Reply:
Since a few releases webui tests also support reboots and go over the entire installation process.